pydocx/openxml/wordprocessing/embedded_object.py

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.models import XmlModel, XmlCollection
+from pydocx.openxml.vml.shape import Shape
+
+
+class EmbeddedObject(XmlModel):
+    """
+    reference: http://www.datypic.com/sc/ooxml/e-w_object-1.html
```

Comment:
I think we should reference https://msdn.microsoft.com/en-us/library/documentformat.openxml.wordprocessing.embeddedobject%28v=office.15%29.aspx instead or additionally. What do you think?